app/services/albums.js

```diff
@@ -0,0 +1,20 @@
+const request_promise = require('request-promise');
+const { logger } = require('express-wolox-logger');
+const api_url = require('../../config').common.albumsApiURL;
+
+const options = {
+  uri: `${api_url}`,
+  headers: {
+    'User-Agent': 'Request-Promise'
+  },
+  json: true
+};
+
+exports.getAlbums = id => {
+  if (id) options.uri = `${api_url}/${id}/photos`;
```

Comment:
what happens if the user wants a specific album? the url would be `api_url/albums/1`